curvefs/src/client/filesystem/filesystem.h

```diff
@@ -79,7 +79,8 @@ class FileSystem {
     CURVEFS_ERROR ReadDir(Request req,
                           Ino ino,
                           FileInfo* fi,
-                          std::shared_ptr<DirEntryList>* entries);
+                          std::shared_ptr<DirEntryList>* entries,
+                          std::string* marker);
```

Comment:
Please don't break the structure of the function, try to look for a more elegant implementation :(

Reply:
ok, I will try